a10_octavia/tests/unit/controller/worker/tasks/test_network_tasks.py

```diff
@@ -14,36 +14,74 @@
 
 
 import mock
+from unittest.mock import patch
 
+from octavia.common import data_models as o_data_models
 from octavia.tests.common import constants as t_constants
-import octavia.tests.unit.base as base
+from octavia.tests.unit import base
 
 from a10_octavia.controller.worker.tasks import a10_network_tasks
 from a10_octavia.network import data_models
 from a10_octavia.network.data_models import Subport
 from a10_octavia.tests.common import a10constants as a10_test_constants
 
-SUBPORT = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID)
+SUBPORT1 = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID1)
+SUBPORT2 = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID2)
+
+VIP = o_data_models.Vip(port_id=t_constants.MOCK_PORT_ID,
+                        subnet_id=t_constants.MOCK_SUBNET_ID,
+                        qos_policy_id=t_constants.MOCK_QOS_POLICY_ID1)
+LB = o_data_models.LoadBalancer(vip=VIP)
+PARENT_PORT = data_models.ParentPort(trunk_id=a10_test_constants.TRUNK_ID)
+EMPTY_PARENT_PORT = data_models.ParentPort()
+DELETE_SUBPORTS = {t_constants.MOCK_AMP_ID1: [SUBPORT1]}
+MULTIPLE_DELETE_SUBPORTS = {t_constants.MOCK_AMP_ID1: [SUBPORT1, SUBPORT2]}
+
+PORT_DELTA = data_models.PortDelta(
+            amphora_id=t_constants.MOCK_AMP_ID1, compute_id=t_constants.MOCK_AMP_COMPUTE_ID1,
+            add_subports=[], delete_subports=[])
 
 
 class TestNetworkTasks(base.TestCase):
 
-    @mock.patch('a10_octavia.controller.worker.tasks.a10_network_tasks.BaseNetworkTask.network_driver')
-    def test_handle_delete_port_deltas(self, get_network_drivers_mock):
-        get_network_drivers_mock.return_value = mock.Mock()
-        mock_parent_port = data_models.ParentPort(trunk_id=a10_test_constants.TRUNK_ID)
+    def setUp(self):
+        patcher = patch(
+            'a10_octavia.controller.worker.tasks.a10_network_tasks.BaseNetworkTask.network_driver')
+        self.network_driver_mock = patcher.start()
+        self.network_driver_mock.return_value = mock.Mock()
```

Comment:
The network driver is a module. It should never be called as a function or class to where a return value is expected. This is not needed.